docs/operation_center/events.md

```diff
@@ -5,3 +5,38 @@ The events page diplays a list of latest events received by SEKOIA.IO. You can s
 On the event page, you can access the raw format of your events, as received by SEKOIA.IO and the STIX format in which it is transformed.
 
 ![Event Page](../assets/operation_center/events.gif)
+
+## Log list:
+The scrolling list displays all the logs you have sent us.
```

Comment:
Not all logs, by default it's the most recents. And there's a maximal log retention in SEKOIA.IO, events older than this parameter won't be searchable or retrived in this page